src/components/Login/stories.tsx

```diff
@@ -8,6 +8,7 @@ storiesOf("Login", module)
   .addDecorator(decoratorCentered)
   .add("default", () => (
     <Router>
-      <Login />
+      {" "}
+      <Login />{" "}
```

Comment:
🤔 what are you achieving with the `{" "}`s?

Reply:
This is due to my linter/Prettier. Below is the rule

Replace `<Router>·<Login·/>·</Router>` with `(⏎····<Router>⏎······{"·"}⏎······<Login·/>{"·"}⏎····</Router>⏎··)`eslint(prettier/prettier)

Reply:
So it won't let you just do this?
 
```tax
<Router>
  <Login />
</Router>
```

Reply:
To me it looks like you might have an extra space between things and prettier is trying to maintain that. If you remove the spaces it may fix that 🤷‍♂